bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -137,7 +137,7 @@ private boolean checkCommitAuthor(List<Commit> commits) throws IOException {
                 var sourceBranch = mergeSourceBranch();
                 if (sourceBranch.isPresent() && sourceRepo.isPresent()) {
                     try {
-                        var mergeSourceRepo = pr.repository().forge().repository(sourceRepo.get());
+                        var mergeSourceRepo = pr.repository().forge().repository(sourceRepo.get()).get();
```

Comment:
Perhaps this (and the other similar ones) could be replaced with `.orElseThrow()` instead? A few are written like this already.

Reply:
Yeah, I agree. Although the effect of a failed `.get()` will be similar (a `NoSuchElementException` will be thrown), I agree that `.orElseThrow()` expresses the intent more clearly to the reader.